lfs-image

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+set +e
+
+# Config options you may pass via Docker like so 'docker run -e "<option>=<value>"':
+# - IMAGE_NAME=<name>, define a static name for your .bin files
+
+cd /opt/nodemcu-firmware
+
+if [ ! -x luac.cross -a !  -x luac.cross.int ]; then
+  echo Error: No cross compiler found. You need to build the firmeware first.
```

Comment:
I've been away from the LFS/luac.cross developments as certainly noticed. Did you recently change the firmware build so that `luac.cross` gets built with it automatically? When I last checked you needed to `make` it separately.

Reply:
You can make it separately if you don't have/want the xtensa toolchain, but it also gets built in a firmware build.